mimic/model/lstm_model.py

```diff
@@ -0,0 +1,80 @@
+from mimic.model.model import Model
+
+from keras.preprocessing.sequence import pad_sequences
+from keras.layers import Embedding, LSTM, Dense, Dropout
+from keras.preprocessing.text import Tokenizer
+from keras.callbacks import EarlyStopping
+from keras.models import Sequential
+import keras.utils as ku 
+import tensorflow as tf
+
+import numpy as np
+import string, os 
+
+class LSTMModel(Model):
+
+    def __init__(self):
+        self.tokenizer = Tokenizer()
+
+    ''' Create sequences to train the model - each is a sequence of tokens that represent characters '''
```

Comment:
You'll also want to revisit this once #14 is merged - `pydocstyle` is quite picky about how you write docstrings.

Reply:
After you get `pipenv` set up, you'll want to run `pipenv run pycodestyle .` and `pipenv run pydocstyle .` - they'll probably complain about a lot of things, because they're quite picky! 😛 I do have Travis CI set up to prevent merging if they complain about anything, so you will have to do that before this can be merged.